takeshape/penny
# More efficient payment method queries

Note: Some low value Storybook stories were removed — only a general subscription view is available in storybook which is... fine.

### Test Plan (Steps to test):

1. This changes the underlying subscription object, so a full regression test on subscriptions is needed...
1. Add a subscription product to your cart (Exceptional Tee)
1. Check out with it to get a new sub in your system.
2. Go to the Account > Subscriptions section.
3. Click through all the actions, e.g., change product options, payment method, delivery date, skip / unskip an order.
4. Cancel the sub to make sure that works.
